assets/js/vendor/flatpickr.min.js

```diff
@@ -0,0 +1,2 @@
+/* flatpickr v4.6.13,, @license MIT */
```

Comment:
:warning: **Warning**: File appears to be minified and cannot be processed (*Internal.Tokenizer.Exception*).